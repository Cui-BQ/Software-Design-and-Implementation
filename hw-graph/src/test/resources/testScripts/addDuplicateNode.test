# Test add duplicate node

# Create a graph
CreateGraph graph1

# Add different nodes
AddNode graph1 n1
AddNode graph1 n2

# Add a duplicate nodes
AddNode graph1 n2

# List its node
ListNodes graph1