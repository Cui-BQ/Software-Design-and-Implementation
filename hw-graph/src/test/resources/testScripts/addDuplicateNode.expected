# Test add duplicate node

# Create a graph
created graph graph1

# Add different nodes
added node n1 to graph1
added node n2 to graph1

# Add a duplicate nodes
failed to add node n2 to graph1

# List its node
graph1 contains: n1 n2